tools/serverpod_cli/lib/src/analyzer/protocol_definition.dart

```diff
@@ -10,9 +10,13 @@ class ProtocolDefinition {
 
   final List<SerializableModelDefinition> models;
 
+  /// The future calls that are a part of this protocol.
+  final List<FutureCallDefinition> futureCalls;
+
   /// Create a new [ProtocolDefinition].
   const ProtocolDefinition({
     required this.endpoints,
     required this.models,
+    this.futureCalls = const [],
```

Comment:
I understand that you probably wanted to avoid the diff that making this required would generate on existing tests, but we should prefer having it as required to avoid errors when using the `ProtocolDefinition` class in the future. It is the same argument as for having both others as required, since there are tests for endpoints that pass `[]` to models and vice-versa.

```suggestion
    required this.futureCalls,
```